### PR TITLE
[utils] Add Unicode Word Break generator

### PR DESCRIPTION
**Pull request by Azoy**
Adds a missing Unicode generator to the repo which generates word break properties.

### CODE REVIEW

**Comment by Azoy** [comment]
@swift-ci please smoke test

**Review by Catfish-Man — APPROVED** [review]
lgtm thank you!

File: utils/gen-unicode-data/Sources/GenWorkBreak/main.swift
Comment (AnthonyLatsis) [inline utils/gen-unicode-data/Sources/GenWorkBreak/main.swift:1]: There seems to be a typo in the file name.

File: utils/gen-unicode-data/Sources/GenWorkBreak/main.swift [hunk 1]
```diff
@@ -0,0 +1,164 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
```
Comment (AnthonyLatsis) [inline utils/gen-unicode-data/Sources/GenWorkBreak/main.swift:5]: 2024?

File: utils/gen-unicode-data/Sources/GenWorkBreak/main.swift
Comment (Azoy) [reply utils/gen-unicode-data/Sources/GenWorkBreak/main.swift:1]: Thanks!

**Comment by Azoy** [comment]
@swift-ci please smoke test

**Pull request merged**